tests/from_csv_test.py

```diff
@@ -0,0 +1,81 @@
+import glob
+import os
+
+import pytest
+import vaex
+
+path = os.path.dirname(__file__)
+csv_path = os.path.join(path, 'data', 'small3.csv')
+
+
+def test_from_csv():
+    _cleanup_generated_files()
```

Comment:
curious why you chose this code pattern, you prefer the files to be around after the test for manual inspection? Otherwise you could do a try/finally? I'm ok with both.

Reply:
At some point of time I spent a lot of time trying to see why all of a sudden the tests stopped passing locally, even reverting the code to `master` did not help. Then it appeared that on one of the previous runs had failed and left some generated files behind. Which made the test start failing even with the correct code.

So in general I don't mind cleaning up in the beginning of the test, to avoid these kind of unnecessary puzzles. But I understand why this might look annoying. Let me just remove it in this case.

Reply:
No, I kind of like it, because it allows you to inspect issues after the test run. Cleanups can be annoying.